modules/uhttp/filters.go

```diff
@@ -55,8 +55,7 @@ type contextFilter struct {
 }
 
 func (f contextFilter) Apply(w http.ResponseWriter, r *http.Request, next http.Handler) {
-	ctx := ulog.NewLogContext(r.Context())
-	next.ServeHTTP(w, r.WithContext(ctx))
+	next.ServeHTTP(w, r)
```

Comment:
In general, I'm unsure why UberFX is now treating loggers differently from other dependencies. To me, the most reasonable thing to do is to use DIG to inject loggers into modules; *that's* the efficient place to add static context like the service name and module name. If you're doing that on the request path, you've already lost the performance game - saving 300ns by storing the logger on the context is penny-wise and pound-foolish.

Separately, I understand that you want to make it easy (or possibly even required) for developers to include span and trace IDs in their logs. I'm all for this - as a simple field type. This is easy because it's *consistent* - tracing information is just like any other field, and doesn't require me to use some special API. It also doesn't require cooperation from my application's middleware, so `ulog` is actually independent of FX's HTTP server.

```
logger.Debugw("failed calling populous", ulog.Trace(ctx))
```